Libraries/Text/Text/RCTTextView.m

```diff
@@ -88,11 +88,6 @@ - (BOOL)canBecomeKeyView
 	return _selectable;
 }
 
-- (BOOL)enableFocusRing
```

Comment:
Interestingly I don't see this method at all upstream, so looks like they already got rid of it.

Reply:
It's a desktop only method

Reply:
... which should probably have a TODO macOS tag now that I think about it. This prop only exists in rn-macos and rn-windows afaik

Reply:
Yea if we added it to a cross plat file then it needs a tag. Otherwise the assumption is FB has that code too

Reply:
This is already within a block

`#if TARGET_OS_OSX // [TODO(macOS GH#774)`